dessia_common/utils/helpers.py

```diff
@@ -72,3 +73,37 @@ def get_python_class_from_class_name(full_class_name: str):
     # Storing in cache
     _PYTHON_CLASS_CACHE[full_class_name] = class_
     return class_
+
+
+def plotdata(selector: str = None, serialize_data: bool = True, load_by_default: bool = False):
```

Comment:
I am puzzled bu the fact we need to set values directly into `function.__dict__`

Reply:
I'll think about another idea to use decorators to do this without set value into `function.__dict__`. Do you have hints or maybe another idea to do this?